README.md

```diff
@@ -25,6 +25,19 @@ Configuration
 
 Nothing special needed.
 
+Releases
+---
+
+Steps to build a new release:
```

Comment:
We should update this with the same language as the other modules we did

Reply:
Done.